docs/tutorials/gcp_cpu.md

```diff
@@ -0,0 +1,242 @@
+
+      1.  Open a browser window.
+      2.  In the navigation bar, paste the URL that you copied in step 4.
+      3.  In the browser you should now see the Jupyter UI, running on your VM.
+
+      The code that you execute here executes on your VM. You can navigate to qsim >
+      docs > tutorials to try other tutorials.
+
+*   {Command line}
+
+      You can run a quantum simulation using qsim from the command line.
+      Your code runs in the qsim Docker container.
+
+      **Before you begin**
+
+      For this scenario, you can connect to your machine directly over SSH rather than
+      create a tunnel. In step 2.3 above, remove the second half of the command.
+      Instead of the following command:
+
+      ```
+      gcloud compute ssh [YOUR_INSTANCE_NAME] -- -L 8888:localhost:8888
+      ```
+
+      Run:
+
+      ```
+      gcloud compute ssh [YOUR_INSTANCE_NAME]
+      ```
+
+      Either command works for the purpose of this tutorial. Continue to step 4 then
+      complete the steps below, regardless of which command you use.
+
+      **1. Copy the container ID for your qsim Docker container**
+
+      Run `docker ps` to display the container ID. The output should look like
+      the following text:
+
+      ```
+      CONTAINER ID IMAGE COMMAND CREATED STATUS PORTS NAMES i
+      8ab217d640a3 gcr.io/quantum-291919/jupyter_qsim:latest "jupyter-notebook --…" 2 hours ago Up 2 hours 0.0.0.0:8888->8888/tcp dazzling_lovelace.
+      ```
+
+      In this case, the container ID is `8ab217d640a3`.
+
+      **2. Connect to your qsim Docker container**
+
+      Run `docker exec` to login to your container. Replace `[CONTAINER_ID]`
+      with the ID that you copied in step 1.
+
+      ```
+      docker exec -it [CONTAINER_ID] /bin/bash
+      ```
+
+      Your command prompt now executes commands in the container.
+
+      **3. Verify your installation**
+
+      You can use the code below to verify that qsim uses your qsim installation.
+      You can paste the code directly into the REPL, or paste the code in a
+      file.
+
+      ```
+      # Import Cirq and qsim
+      import cirq
+      import qsimcirq
+
+      # Instantiate qubits and create a circuit
+      q0, q1 = cirq.LineQubit.range(2)
+      circuit = cirq.Circuit(cirq.H(q0), cirq.CX(q0, q1))
+
+      # Instantiate a simulator that uses the GPU
+      qsim_simulator = qsimcirq.QSimSimulator()
+
+      # Run the simulation
+      print("Running simulation for the following circuit:")
+      print(circuit)
+
+      qsim_results = qsim_simulator.compute_amplitudes(
+          circuit, bitstrings=[0b00, 0b01])
+
+      print("qsim results:")
+      print(qsim_results)
+      ```
+
+      After a moment, you should see a result that looks similar to the following.
+
+      ```
+      [(0.7071067690849304+0j), 0j]
+      ```
+
+## Next steps
+
+After you finish, don't forget to stop or delete your VM on the Compute
+Instances dashboard.
```

Comment:
This should be emphasized and include a note about billing (e.g. end with "...to prevent further billing.")

Reply:
Fixed.